### ISSUE TITLE
word-count: Test for handling of multiple whitespace

### ISSUE THREAD

**Issue by jonmcalder** [exercism/problem-specifications#1022]
There is a suggestion in [exercism/r #85](https://github.com/exercism/r/issues/85) that this exercise should include a test for multiple whitespace (i.e. check that words are trimmed of whitespace and that repeated whitespace isn't treated as another "word").

This could be done either by adding a simple new test case, or via adapting the inputs for an existing test case e.g. 
```
"description": "ignore punctuation",
      "property": "countwords",
      "input": " car : carpet as java :  javascript  !!&@$%^&",       # note extra spaces here
      "expected": {
        "car": 1,
        "carpet": 1,
        "as": 1,
        "java": 1,
        "javascript": 1
      }
```

Was this type of test case omitted deliberately in order to simplify the exercise? If not, this seems to me like it could be a worthwhile addition to the canonical test data.

**Comment by Insti** [comment]
[Description](https://github.com/exercism/problem-specifications/blob/master/exercises/word-count/description.md)

Adding a new test case which contains a specific example of this is a good idea.
The input text should be the simplest possible example that will trigger this bug.
The example provided in the linked issue looks fine.

**Comment by jonmcalder** [comment]
Cool - will do so a little later.

**Comment by ErikSchierboom** [comment]
This has been closed by #1023.
